subscriptions/tests/test_models.py

```diff
@@ -25,3 +30,38 @@ def test_enrolling_deletes_same_event_free_spot_subscriptions(
     EnrolmentFactory(occurrence=occurrence_2, child=child)
 
     assert_subscriptions(child, other_event_subscription)
+
+
+@pytest.mark.django_db
+def test_enrolling_deletes_same_event_group_free_spot_subscriptions(child):
+    event_group = EventGroupFactory(published_at=now())
+    event = EventFactory(published_at=now(), event_group=event_group)
+    same_group_event = EventFactory(published_at=now(), event_group=event_group)
+    other_group_event = EventFactory(
+        published_at=now(), event_group=EventGroupFactory()
+    )
+
+    occurrence = OccurrenceFactory.create(time=now() + timedelta(days=1), event=event)
+    same_event_occurrence = OccurrenceFactory.create(
+        time=now() + timedelta(days=2), event=event
+    )
+    same_group_occurrence = OccurrenceFactory(
+        time=now() + timedelta(days=3), event=same_group_event
+    )
+    other_group_occurrence = OccurrenceFactory(
+        time=now() + timedelta(days=4), event=other_group_event
+    )
+
+    same_event_subscription = FreeSpotNotificationSubscriptionFactory(  # noqa: F841
+        child=child, occurrence=same_event_occurrence
+    )
+    same_group_subscription = FreeSpotNotificationSubscriptionFactory(  # noqa: F841
+        child=child, occurrence=same_group_occurrence
+    )
+    other_group_subscription = FreeSpotNotificationSubscriptionFactory(
+        child=child, occurrence=other_group_occurrence
+    )
+
+    EnrolmentFactory(child=child, occurrence=occurrence)
+
+    assert_subscriptions(child, other_group_subscription)
```

Comment:
Since I am still an outsider, I had to explore what this is doing. **Trivial** thing, but I would add comments on these lines so it would be clear what is being asserted. Or change the assert_subscriptions -function name, because it's not clear that assert_subscriptions -function compares child subscriptions to this other set of subscriptions.

Only a trivial thing, so can be approved like this also!